src/minimalin.c

```diff
@@ -153,11 +154,18 @@ static void update_current_time() {
   const time_t temp = time(NULL);
   const struct tm *tick_time = localtime(&temp);
   int hour = tick_time->tm_hour;
+  int minutes = tick_time->tm_min;
+  bool should_vibrate_on_the_hour = config_get_bool(s_config, ConfigKeyVibrateOnTheHour);
   if(hour > 12){
     hour -= 12;
   }else if(hour == 0){
     hour = 12;
   }
+
+  if(should_vibrate_on_the_hour && minutes == 0){
+    vibes_short_pulse();
```

Comment:
It would be better to move this in the `tick_handler` function. Something like:

```diff
 static void tick_handler(struct tm *tick_time, TimeUnits units_changed){
+  if(units_changed & HOUR_UNIT){
+    bool vibrate_on_the_hour = config_get_bool(s_config, ConfigKeyVibrateOnTheHour);
+    if(vibrate_on_the_hour){
+      vibes_short_pulse();
+    }
+  }
   schedule_weather_request(10000);
```

That way we also avoid calling the config if the hour didn't change.